scheduler/scheduler-server/src/test/java/functionaltests/utils/SchedulerTestConfiguration.java

```diff
@@ -38,6 +38,8 @@
 
     public static final SchedulerTestConfiguration NOT_STARTED = new SchedulerTestConfiguration();
 
+    public static final int TEST_TIMEOUT = 2100000;
```

Comment:
Can you comment to which time this value corresponds to?

Reply:
Comment added. It deals with a timeout variable that replaces 'CentralPAPropertyRepository.PA_TEST_TIMEOUT.getValue()'. Hence, it avoids using a variable in a separate project (i.e., programming) to configure the timeout of scheduler tests